site/layouts/wrapper/single.html

```
@@ -0,0 +1,29 @@
+{{/*
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)